controllers/solrprometheusexporter_controller.go

```diff
@@ -216,6 +216,11 @@ func (r *SolrPrometheusExporterReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 				}
 			}
 		}
+	} else if prometheusExporter.Spec.SolrReference.SolrTLS != nil && prometheusExporter.Spec.SolrReference.SolrTLS.MountedTLSDir != nil {
+		tlsClientOptions = &util.TLSClientOptions{}
+		tlsClientOptions.TLSOptions = prometheusExporter.Spec.SolrReference.SolrTLS
+		// TODO: How to get this from the SolrCloud reference?
```

Comment:
Would appreciate some advice on how best to handle this? Maybe just add the same:
```
	// +optional
	BusyBoxImage *ContainerImage `json:"busyBoxImage,omitempty"`
```
For the Prometheus exporter CRD?

Reply:
Yeah, no other option 😢 

Maybe in the description of the field we can say it is only used when mountedTLS is enabled?

Reply:
SGTM ;-) Will update shortly